src/main/java/org/cryptomator/ui/vaultoptions/GeneralVaultOptionsController.java

```diff
@@ -47,6 +48,8 @@ public void initialize() {
 		actionAfterUnlockChoiceBox.getItems().addAll(WhenUnlocked.values());
 		actionAfterUnlockChoiceBox.valueProperty().bindBidirectional(vault.getVaultSettings().actionAfterUnlock());
 		actionAfterUnlockChoiceBox.setConverter(new WhenUnlockedConverter(resourceBundle));
+		lockAfterTimeCheckbox.selectedProperty().bindBidirectional(vault.getVaultSettings().autoLockWhenIdle());
+		Bindings.bindBidirectional(lockTimeInMinutesTextField.textProperty(), vault.getVaultSettings().autoLockIdleSeconds(), new IdleTimeSecondsConverter());
```

Comment:
Looks like this PR is polluted with changes already merged in #1603. Please make sure to avoid such noise.

Reply:
Oh I see. Health Check doesn't provide a laeger number of options, to be precise. On the other hand, autolock might grow in the future.

But that is ok for now. Let's deal with it when the time comes.

Remove the unused stopwatch glyph, though!

Reply:
Fixed in 2d4a256